src/Consolonia.Gallery/Gallery/GalleryViews/GalleryScrollViewer.axaml.cs

```diff
@@ -66,7 +66,7 @@ public ScrollViewerPageViewModel()
                                       `^'     \ :/           `^'  `-^-'   \v/ :  \/ 
                               Bill Ames
 
-                              """.Split(['\r', '\n'], StringSplitOptions.RemoveEmptyEntries));
+                              """.Split((char[])['\r', '\n'], StringSplitOptions.RemoveEmptyEntries));
```

Comment:
I'm curios what was the type before cast

Reply:
super weird. I'm using my ARM tablet and build is broken on ARM because it is ambigious between 
```
.Split(char[], ...)
.Split(string, ...)
```
methods. 
I think this is a dotnet bug in the arm implementation or something.  I couldn't build without explicitely casting.

Reply:
I think string has char[] cast operator or something.

Reply:
very interesting. ARM64? what is the device?